components/brave_new_tab_ui/components/default/braveToday/customize/FeedCard.tsx

```diff
@@ -28,6 +33,12 @@ const Card = styled('div').attrs<CardProps>(props => ({
   border-radius: 8px;
   overflow: hidden;
   box-shadow: 0px 0px 16px 0px #63696E2E;
+
+  &[data-feed-card-is-followed=true] {
+    &:not(:hover, :focus-within) ${StyledFollowButton} {
+      opacity: 0;
```

Comment:
Same here, it just feels a bit odd. Maybe this is what we want though.

Reply:
Fixed